cherab/solps/formats/balance.py

```diff
@@ -94,7 +94,7 @@ def load_solps_from_balance(balance_filename):
             for key in fhandle.variables.keys():
                 if 'fna_' in key:
                     fna += fhandle.variables[key].data.copy()
-            if fna is not 0:
+            if fna != 0:
```

Comment:
Please, replace with:
```
if np.any(fna != 0):
```

Reply:
Yep, just saw this when I tried to run these examples. Thanks.